portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,110 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Kayla Leung Portfolio</title>
+    <title>Kayla Leung</title>
+
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font.css">
```

Comment:
font.css does not exist at this commit, was this meant for later?

Reply:
Most recent commit should now have the font.css file included.